app/frontend/package.json

```diff
@@ -35,16 +35,20 @@
     "bootstrap-scss": "^4.6.1",
     "core-js": "^3.23.3",
     "deep-freeze": "^0.0.1",
+    "export-from-json": "^1.7.2",
```

Comment:
What is the purpose of this package "export-from-json": "^1.7.2"

Reply:
We dont use this package any more we gonna  remove it